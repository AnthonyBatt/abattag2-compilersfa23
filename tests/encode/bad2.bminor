"G isnt a hex value, so this will error: \0xgG"
