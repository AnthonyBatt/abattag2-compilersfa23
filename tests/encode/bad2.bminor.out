invalid hexdigits given, valid hexdigits are 0-9, a-f, and A-F, given digits were: g and G
decode("G isnt a hex value, so this will error: \0xgG"): the string was unable to be decoded
